emr-dynamodb-hadoop/src/main/java/org/apache/hadoop/dynamodb/filter/DynamoDBQueryFilter.java

```diff
@@ -38,6 +38,11 @@ public DynamoDBQueryFilter withKeyCondition(String key, Condition condition) {
     return this;
   }
 
+  public DynamoDBQueryFilter withScanFilter(String key, Condition condition) {
+    scanFilter.put(key, condition);
```

Comment:
What will happen if condition already exists for that key? What would be expected behavior?